cartography/util.py

```diff
@@ -297,3 +302,47 @@ def batch(items: Iterable, size: int = DEFAULT_BATCH_SIZE) -> List[List]:
         items[i: i + size]
         for i in range(0, len(items), size)
     ]
+
+
+def to_asynchronous(func: Callable[..., R], *args: Any, **kwargs: Any) -> Awaitable[R]:
+    '''
+    Returns a Future that will run a function in the default threadpool.
+    Helper until we start using pytohn 3.9's asyncio.to_thread
+
+    Calls are also wrapped within a backoff decorator to handle throttling errors.
+
+    example:
```

Comment:
Good that you included an example, but since this is going to be a core function, I think we should have detailed docs on what each of the params is.

Reply:
a

Reply:
added